externals/kyuubi-hive-sql-engine/src/main/scala/org/apache/kyuubi/engine/hive/session/HiveSessionImpl.scala

```diff
@@ -83,3 +96,244 @@ class HiveSessionImpl(
     }
   }
 }
+
+object HiveSessionImpl {
+  // scalastyle:off line.size.limit
+  // From https://docs.microsoft.com/en-us/sql/t-sql/language-elements/reserved-keywords-transact-sql#odbc-reserved-keywords
+  // scalastyle:on line.size.limit
+  final val ODBC_KEYWORDS = Sets.newHashSet(
```

Comment:
spotless prefer this code style.

Reply:
why should we provide such a static keyword list?

Reply:
In fact, want to be consistent with [Hive behavior](https://github.com/apache/hive/blob/master/service/src/java/org/apache/hive/service/cli/session/HiveSessionImpl.java#L508), But it's okay to expose all of it.

Reply:
I see, we should preserve it